README.md

```diff
@@ -74,3 +74,18 @@ In case there is an unexpected issue with this process you can test this process
 ## Learn More
 
 For additional information about the Bridges Project please refer to [parity-bridges-common](https://github.com/paritytech/parity-bridges-common) repository.
+
```

Comment:
Personally i would not add this on the documentation until it is moved to `paritytech` hub .

Reply:
This is parity open-source property and I provide the credentials. Why would not add that?

Reply:
What i'm pointing out is to avoid sharing the docker image that resides in your personal docker profile: `https://hub.docker.com/repository/docker/wirednkod13/parity-bridges-ui`

I would wait to announce this docker image for th UI once it resides in the `paritytech` hub.
Let's suppose that in the short term you make any changes on your profile that makes the image unavailable.